sample.env

```diff
@@ -5,8 +5,8 @@ BLOCK_DATA_SOURCE=MINAEXPLORER
 
 # Pool Configuration
 COMMISSION_RATE=.05
-MF_COMMISSION_RATE=0.05
-O1_COMMISSION_RATE=0.05
+MF_COMMISSION_RATE=0.08
+O1_COMMISSION_RATE=0.08
```

Comment:
O1 COMMISSION RATE should be: 0.05
https://blog.o1labs.org/o-1-labs-delegation-policy-786bf96f9fdd

Reply:
thanks - fixed

Reply:
updated